packages/config/config/devices/0x0371/zw175.json

```diff
@@ -6,295 +6,234 @@
 					"value": 1
 				},
 				{
-					"label": "Report Binary Switch",
+					"label": "Turn off",
 					"value": 2
+				},
+				{
+					"label": "Cyclce on/off in 5 second intervals",
+					"value": 3
 				}
 			]
 		},
-		"81": {
-			"label": "LED indicator mode",
-			"description": "Allows you to configure how the LED indicator works",
-			"valueSize": 1,
+		"10": {
+			"label": "Method to Disable Alarm",
+			"description": "Allowable range: 10-255 - defines alarm duration",
+			"valueSize": 2,
+			"unit": "seconds",
 			"minValue": 0,
-			"maxValue": 2,
-			"defaultValue": 2,
-			"allowManualEntry": false,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
 			"options": [
 				{
-					"label": "LED Disabled",
+					"label": "Tap action button 3x",
 					"value": 0
 				},
 				{
-					"label": "Night Light Mode",
+					"label": "Idle state from corresponding alarm",
 					"value": 1
-				},
-				{
-					"label": "On/Off Mode",
-					"value": 2
 				}
 			]
 		},
+		"18": {
+			"label": "LED Blinking Frequency",
+			"description": "Defines the number of blinks per second during alarm activation",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 9,
+			"defaultValue": 2,
+			"unsigned": true
+		},
+		"19": {
+			"$import": "../0x0086/templates/aeotec_template.json#blink_duration"
+		},
+		"20": {
+			"$import": "../templates/master_template.json#state_after_power_failure_prev_on_off"
+		},
+		"80": {
+			"$import": "../0x0086/templates/aeotec_template.json#binary_report_type"
+		},
+		"81": {
+			"$import": "../0x0086/templates/aeotec_template.json#led_indicator_three_options_alt"
+		},
 		"82[0xff000000]": {
-			"label": "Night Light enable (hour)",
-			"description": "Enable or disable Night Light Mode during specific times",
+			"label": "Night Light (Enable): Hour",
+			"description": "Allowable range: 0-23",
 			"valueSize": 4,
 			"minValue": 0,
-			"maxValue": 24,
+			"maxValue": 23,
 			"defaultValue": 18
 		},
 		"82[0xff0000]": {
-			"label": "Night Light enable (minute)",
-			"description": "Enable or disable Night Light Mode during specific times",
+			"label": "Night Light (Enable): Minute",
+			"description": "Allowable range: 0-59",
 			"valueSize": 4,
 			"minValue": 0,
-			"maxValue": 60,
+			"maxValue": 59,
 			"defaultValue": 0
 		},
 		"82[0xff00]": {
-			"label": "Night Light disable (hour)",
-			"description": "Enable or disable Night Light Mode during specific times",
+			"label": "Night Light (Disable): Hour",
+			"description": "Allowable range: 0-23",
 			"valueSize": 4,
 			"minValue": 0,
-			"maxValue": 24,
+			"maxValue": 23,
 			"defaultValue": 8
 		},
 		"82[0xff]": {
-			"label": "Night Light disable (minute)",
-			"description": "Enable or disable Night Light Mode during specific times",
+			"label": "Night Light (Disable): Minute",
+			"description": "Allowable range: 0-59",
 			"valueSize": 4,
 			"minValue": 0,
-			"maxValue": 60,
+			"maxValue": 59,
 			"defaultValue": 0
 		},
```

Comment:
Just to make sure - this isn't the weird time as hex thing, right?

Reply:
Correct, this one the manual says the value per bit. I removed the "last setting" ones as they don't make much sense for bitmasks.

"Enable Hour:
Valid value are 0-23 and 127. Value 127 means use last valid setting.
Enable Minute:
Valid value are 0-59 and 255. Value 255 means use last valid setting"